tox.ini

```diff
@@ -141,6 +141,18 @@ commands=
     {envpython} runtests_setup.py build --build-exe build
     {envpython} tox_run.py
 
+[testenv:coveralls]
```

Comment:
I see you are adding coveralls results as well... can we get a badge for that too, please? :smile: